test/e2e/lifecycle/upgrade_test.go

```diff
@@ -32,6 +33,7 @@ var _ = Context("Cluster Network Addons Operator", func() {
 				BeforeEach(func() {
 					UninstallRelease(oldRelease)
 					InstallRelease(newRelease)
+					CheckCrdExplainable()
```

Comment:
Would it make sense to keep the second one in `It`? Since it is really something we want to test, not a setup of the context